tests/scripts/installandtest.sh

```diff
@@ -54,6 +54,10 @@ fi
 
 
 
+#!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+# DO NOT MERGE THIS COMMIT
```

Comment:
Is this a problem for this PR? Or do we not intend to merge this?

Reply:
That is a comment for the noop sleep below so I have enough time to debug the CI environment in realtime.  I'm trying to determine why it is failing in CI to prevent this CI test from blocking other CI runs after merging

Reply:
Looks like this might be an issue with the latest update to the ods-ci automation repo. Testing this locally to resolve the issue

Reply:
This `sleep` has been removed.  I updated the CI test to add some custom code as a workaround for the latest version of ods-ci